### ISSUE TITLE
Using angular-cron-gen in modal, week tab layout failure

### ISSUE THREAD

**Issue by FritzHerbers** [vincentjames501/angular-cron-gen#3]
Noticed a small flaw when using angular-cron-gen in a modal. The weekdays in the week tab are not grid-like positioned.

![2016-11-23_162229](https://cloud.githubusercontent.com/assets/10029682/20568244/2db92522-b19c-11e6-96ec-d07dec909750.png)

Tried also with container and container-fluid in modal.

I think of the inproper use of http://getbootstrap.com/css/#forms-controls-static

```
<script type="text/ng-template" id="wfEditCronExpressionHelper.html">
                            <div class="modal-header">
                                <h3 class="modal-title custom-font">Cron Expression Helper</h3>
                            </div>
                            <div class="modal-body">
                                <div class="row">
                                    <div>
                                        <cron-gen ng-model="vm.editCronExpressionHelperCronExpression" options="vm.editCronExpressionHelperCronOptions"></cron-gen>
                                    </div>
                                </div>
                            <div class="modal-footer">
                                <button class="btn btn-success btn-ef btn-ef-3 btn-ef-3c" ng-click="ok()"><i class="fa fa-arrow-right"></i>OK (use expression)</button>
                                <button class="btn btn-lightred btn-ef btn-ef-4 btn-ef-4c" ng-click="cancel()"><i class="fa fa-arrow-left"></i>Cancel</button>
                            </div>
                        </script>
```

```
        vm.editCronExpressionHelper = function () {

            var modalInstance = $uibModal.open({
                templateUrl: 'wfEditCronExpressionHelper.html',
                controller: 'SimpleItemModalInstanceCtrl',
                scope: $scope,
                size: "lg",
            });

            modalInstance.result.then(function () {
                // something
            }, function () {
            });
        };
```

**Comment by vincentjames501** [comment]
Yeah you're probably right. PRs welcome :). If I don't get one I'll fix then when I get back from vacation.

**Comment by vincentjames501** [comment]
Fixed in da7ae4933bc6548da5095e1a33d9c36a58976a50
